From a5f506ada43fa009e3c425172f94ce2bc42d4f66 Mon Sep 17 00:00:00 2001
From: Karl Mehltretter <kmehltretter@gmail.com>
Date: Sat, 22 Aug 2026 14:16:51 +0200
Subject: docs: debugobjects: Fix debug_object_init() name
Git-commit: a5f506ada43fa009e3c425172f94ce2bc42d4f66
Patch-mainline: v7.4-rc1
References: git-fixes

The initialization helper is debug_object_init(), not
debug_objects_init().

Fixes: 691cc54c7d28 ("debugobjects: add documentation")
Assisted-by: Codex:gpt-5.6-sol
Signed-off-by: Karl Mehltretter <kmehltretter@gmail.com>
Reviewed-by: Randy Dunlap <rdunlap@infradead.org>
Signed-off-by: Jonathan Corbet <corbet@lwn.net>
Message-id: <20260822121651.1559-8-kmehltretter@gmail.com>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 Documentation/core-api/debug-objects.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/core-api/debug-objects.rst b/Documentation/core-api/debug-objects.rst
index ac926fd55a64..708775fc581b 100644
--- a/Documentation/core-api/debug-objects.rst
+++ b/Documentation/core-api/debug-objects.rst
@@ -129,7 +129,7 @@ When the real object is not yet tracked by debugobjects then the
 fixup_activate function is called if available. This is necessary to
 allow the legitimate activation of statically allocated and initialized
 objects. The fixup function checks whether the object is valid and calls
-the debug_objects_init() function to initialize the tracking of this
+the debug_object_init() function to initialize the tracking of this
 object.
 
 When the activation is legitimate, then the state of the associated
-- 
2.55.0

